engine/src/main/resources/org/operaton/bpm/engine/impl/mapping/entity/Resource.xml

```diff
@@ -58,11 +58,8 @@
     delete from ${prefix}ACT_GE_BYTEARRAY where DEPLOYMENT_ID_ = #{id}
   </delete>
 
-  <delete id="purgeTablePreserveLicenseKey" parameterType="string">
+  <delete id="purgeTable" parameterType="string">
```

Comment:
Here is a complete new functionality now. This will purge the whole ACT_GE_ByteArray Table. Is this something which we want?

@javahippie can you say what do you think here?

Reply:
I think this functionality purged the whole table before, too, but left the rows which stored a license key in place, didn't it?

Reply:
Ok, got it. So we can let it like it is now. :)